### ISSUE TITLE
Add "Text" marks

### ISSUE THREAD

**Issue by jwildfire** [RhoInc/Webcharts#14]
Add functionality for drawing marks that render svg text. Positioning the marks should be very similar to the settings for "circle" marks, but not sure the best way to specify the text content. Maybe the same system used to specify tooltips now?

**Comment by nbryant** [comment]
```javascript
{
  type: 'text',
  per: ['site'],
  text: '$y',
  summarizeY: 'count',
  attributes: {
    dx: '0.25em'
  }
}
```

**Comment by nbryant** [comment]
addressed in [v1.6.0](https://github.com/RhoInc/Webcharts/releases/tag/v1.6.0)
